README.md

```diff
@@ -1,2 +1,2 @@
-# clevertap-directcall-flutter-sdk
-Flutter SDK for CleverTap Direct Call
+# clevertap-signedcall-flutter-sdk
+Flutter SDK for CleverTap Signed Call
```

Comment:
@shivamsharma2710 Need a better ReadMe for this

Reply:
@darshanclevertap Yes. Will work on this.

Reply:
@darshanclevertap This is resolved!